src/docs/background/13-how-payment-intents-work/index.md

```diff
@@ -0,0 +1,187 @@
+marks the payment as preauthorized in Flex.
+
+### stripe-capture-payment-intent
+
+Captures a confirmed PaymentIntent.
+
+### stripe-refund-payment
+
+Either cancels an unconfirmed PaymentIntent or refunds the related captured
+charge.
+
+## Required actions in the client
+
+The required actions in the client are related to authentication and
+confirmation. You need to be able to handle potential authentication steps
+required by the customer's card issuing bank. After authentication, the client
+needs to [obtain PaymentIntent data from the transaction's protected
+data](#implementing-the-automatic-paymentintent-flow) and use that to confirm
+the payment.
+
+In case you wish to enforce [3D Secure Card
+Payments](https://stripe.com/docs/payments/3d-secure) for cards that support
+3DS, in addition to supporting payment authentication in your client app, you
+may need to update your [Stripe Radar
+rules](https://stripe.com/docs/payments/3d-secure#three-ds-radar).
+
+### Handling Strong Customer Authentication
+
+[Strong Customer
+Authentication](https://stripe.com/en-fi/payments/strong-customer-authentication)
+is a potential step enforced by governmental regulation. Not every PaymentIntent
+will require customer authentication. For instance, authentication may not be
+required for:
+
+- transactions out of scope of SCA
+- e.g. when card issuing bank is outside of EEA
+- merchant initiated transactions
+- transactions that fall under an SCA exemption
+- low value or low risk transactions
+- recurring payments for fixed amount
+- other
+
+This means that Flex implementation of PaymentIntents supports payment flows
+that require authentication and those that do not. When implementing the
+PaymentIntent flow in the client you need to be prepared for handling both
+cases - payments requiring SCA and payments that do not. It might be impossible
+to know in advance whether the payment will require authentication, unless the
+marketplace and all its customers are outside of EEA.
+
+The recommended way of implementing support for SCA is to use [Stripe
+Elements](https://stripe.com/docs/payments/payment-intents/quickstart) that can
+provide you with ready modals for handling e.g. 3D Secure Card Payments. The
+next chapter will provide high level instructions on how to do this in the
+client.
+
+### Implementing the automatic PaymentIntent flow
+
+[Stripe Elements](https://stripe.com/docs/payments/payment-intents/quickstart)
+provides ready tools and a reference for implementing the automatic
+PaymentIntent flow. It's the recommended way to support PaymentIntents in the
+client and handle the authentication steps.
+
+For implementing the PaymentIntent flow, you need to follow the steps in
+[Automatic confirmation
+quickstart](https://stripe.com/docs/payments/payment-intents/quickstart#automatic-confirmation-flow).
+With Flex, the first step differs from Stripe instructions, since that is
```

Comment:
Maybe this could be clearer if it was just stated that Flex handles this step in the mentioned action. So something like:

_With Flex, the first step in handled by the transaction engine when a process transition that has the `stripe-create-payment-intent` action is transitioned._

Or something similar on a higher level if that feels like a too detailed description. However, now that we're planning to enable process editing with the CLI, at some point we need to get quite detailed with the docs.